web/tests/fork-workspace.spec.ts

```diff
@@ -8,11 +8,11 @@ let workspace: Workspace;
 test('fork workspace', async function ({ browser }) {
     splash = await Splash.open(browser);
     workspace = await splash.openNewWorkspace('test-example');
-    await workspace.page.pause();
     await workspace.page.locator('#save-workspace-as-starter-button').click();
-    await workspace.page.locator('#save-workspace-as-input').locator('input').fill('test-example-fork');
-    await workspace.page.locator('#save-workspace-as-input').locator('#ok').click();
-    await workspace.page.waitForURL('**/#/workspace/test-example-fork');
+    await workspace.page.locator('#save-workspace-as-input input').fill('automated-tests/subdirectory/test-example-fork');
```

Comment:
Instead of including `automated-tests` here, could you please use `ROOT` from `lynxkite.ts`? There's no big practical reason for it, but that's what we did in all the other files. Thanks.